cockatrice/src/abstractcarditem.cpp

```diff
@@ -80,6 +82,9 @@ void AbstractCardItem::transformPainter(QPainter *painter, const QSizeF &transla
     int fontSize = round(translatedSize.height() / 8);
     if (fontSize < 9)
         fontSize = 9;
+    if (fontSize > MAX_FONT_SIZE)
+        fontSize = MAX_FONT_SIZE;
```

Comment:
You can consolidate all these lines with

```cpp
const int fontSize = std::max(9, std::min(MAX_FONT_SIZE, round(translatedSize.height() / 8)));
```

The reason to prefer the one-liner is because you can make the int const and the rest of the function can't accidentally modify it. By making things immutable and declarative it's easier to reason about at a larger scope.